--- rpi3_common.c.old	2023-11-24 18:13:48.090781736 +1300
+++ rpi3_common.c	2023-11-24 18:14:10.247562026 +1300
@@ -11,7 +11,7 @@
 #include <debug.h>
 #include <interrupt_mgmt.h>
 #include <platform_def.h>
-#include <uart_16550.h>
+#include <pl011.h>
 #include <xlat_tables_v2.h>
 
 #include "rpi3_hw.h"
@@ -84,14 +84,14 @@
 /*******************************************************************************
  * Function that sets up the console
  ******************************************************************************/
-static console_16550_t rpi3_console;
+static console_pl011_t rpi3_console;
 
 void rpi3_console_init(void)
 {
-	int rc = console_16550_register(PLAT_RPI3_UART_BASE,
-					PLAT_RPI3_UART_CLK_IN_HZ,
-					PLAT_RPI3_UART_BAUDRATE,
-					&rpi3_console);
+	int rc = console_pl011_register(PLAT_RPI3_UART_BASE,
+					    PLAT_RPI3_UART_CLK_IN_HZ,
+					    PLAT_RPI3_UART_BAUDRATE,
+					    &rpi3_console);
 	if (rc == 0) {
 		/*
 		 * The crash console doesn't use the multi console API, it uses
